### PR TITLE
release cleanup

### PR DESCRIPTION
**Pull request by loriab**
## Todos
- [x] few more copyrights.
- [x] remove unused testing scripts.

## Status
- [ ] Ready for review
- [ ] Ready for merge

### CODE REVIEW

File: psi4/driver/p4util/python_helpers.py [hunk 1]
```diff
@@ -195,8 +195,8 @@ def _core_wavefunction_from_file(wfn_data):
         wfn_dimension[label] = core.Dimension.from_list(tup, name=label) if tup is not None else None
 
     for label in wfn_matrixarr:
-        array = wfn_dimension[label]
-        wfn_dimension[label] = core.Matrix.from_array(array, name=label) if array is not None else None
+        array = wfn_matrixarr[label]
```
Comment (dgasmith) [inline psi4/driver/p4util/python_helpers.py]: Surprised this passed tests.

Comment (loriab) [reply psi4/driver/p4util/python_helpers.py]: only hits when there's array psivars. they're pretty rare.

**Comment by susilehtola** [comment]
I'm confused: `samples/scf-guess` appears to contain the same calculations and tests as `tests/scf-guess`.

**Comment by loriab** [comment]
In samples one has TEST lines filtered away and test has lines remaining.

**Pull request merged**